builder/builder.go

```diff
@@ -144,7 +144,14 @@ func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allU
 		return err
 	}
 	b.UpstreamURL = upstreamURL
-	b.UpstreamBundlesURL = bundlesURL
+
+	if upstreamBundlesURL != "" {
+		b.UpstreamBundlesURL = upstreamBundlesURL
+		b.Config.Swupd.UpstreamBundlesURL = upstreamBundlesURL
+	}
+	if err := b.Config.SaveConfig(); err != nil {
+		return errors.Wrapf(err, "Error while saving builder config file")
+	}
```

Comment:
Following my suggesting from my previous commit, this won't be needed. besides, this code just overwrites the value set by the user in the config, it doesn't work.